packages/storybook/stories/va-modal-uswds.stories.tsx

```diff
@@ -17,8 +17,50 @@ export default {
   parameters: {
     componentSubtitle: 'va-modal web component',
     docs: {
-      page: () => <StoryDocs storyDefault={Default} data={modalDocs} />,
+      page: () => {
+        // Use effect to control scroll position
+        useEffect(() => {
+          // Function to scroll to top
+          const scrollToTop = () => window.scrollTo(0, 0);
+          
+          // Immediate scroll
+          scrollToTop();
+          
+          // Schedule multiple scroll attempts with increasing delays
+          // This helps ensure scrolling works even if Storybook does additional rendering
+          const timers = [50, 100, 300, 500, 1000].map(delay => 
+            setTimeout(scrollToTop, delay)
+          );
```

Comment:
Internally, `va-modal` does some scrolling and focusing when the modal is opened. The modals aren't supposed to start out in this open state so I'm having to work against that behavior a little bit especially since there are multiple modal components on the Storybook page.